make validate fails: pyproject version (0.0.0) does not match latest tag (v0.0.1)

## Finding
`make validate` (rhiza structure tests) is RED on `main`.

```
.rhiza/tests/structure/test_pyproject.py::TestGitTagVersion::test_latest_tag_matches_pyproject_version
AssertionError: Latest git tag 'v0.0.1' (-> '0.0.1') does not match [project].version '0.0.0' in pyproject.toml
```

The latest released git tag is `v0.0.1`, but `pyproject.toml` still declares `[project].version = "0.0.0"`. The Rhiza structure test `TestGitTagVersion` requires the latest `vX.Y.Z` tag to match the static `[project].version`.

## Subcategory
Build/release correctness (validate gate): current 0 (gate red) -> target 10 (gate green).

## Scope
Locally-owned: `pyproject.toml` line 3.

## Done when
- `[project].version` in `pyproject.toml` equals the latest git tag (`0.0.1`).
- `make validate` passes.

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,6 +1,6 @@
 [project]
 name = "cvxmarkowitz"
-version = "0.0.0"
+version = "0.0.1"
 description = "Markowitz"
 authors = [{name = "Thomas Schmelzer", email = "thomas.schmelzer@gmail.com"},
     {name = "Kasper Johansson"},

